source/javascripts/components/GuidedOnboarding/Collapsible/Collapsible.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { useState, useRef, useEffect, ReactNode } from "react";
+import { Icon, Text, Flex } from "@bitrise/bitkit";
+import "./Collapsible.scss";
+interface CollapsibleProps {
+  children: ReactNode;
+  open?: boolean;
+  title: string;
+}
+
+/**
+ * The usage of this component can be replaced with Accordion in latest Bitkit.
+ */
+const Collapsible = ({ open = false, children, title }: CollapsibleProps): JSX.Element => {
+  const [isOpen, setIsOpen] = useState(open);
+  const [contentHeight, setContentHeight] = useState<number | undefined>(
+    open ? undefined : 0
+  );
+  const ref = useRef<HTMLDivElement>(null);
+
+  useEffect(() => {
+    if (!contentHeight || !isOpen || !ref.current) return;
+    const resizeObserver = new ResizeObserver((contentElement: ResizeObserverEntry[]) => {
+      setContentHeight(contentElement[0].contentRect.height);
+    });
+    resizeObserver.observe(ref.current);
```

Comment:
Normally I would say pls use callback refs for these things but this component will be switched to the one in bitkit so I guess it is ok.

Reply:
ok, will leave as is for now